src/app/components/drawer/base-drawer.tsx

```diff
@@ -1,6 +1,7 @@
 import { ReactNode, Suspense, memo, useCallback, useRef } from 'react';
 import { useNavigate } from 'react-router-dom';
 
+import { Dialog } from '@radix-ui/themes';
 import { Flex, FlexProps, transition, useEventListener } from '@stacks/ui';
```

Comment:
It would be good if we can remove this `@stacks/ui` stuff while we're here. `Flex` anyway